### DISCUSSION TITLE
Residual errors and 2nd order IFCs

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#187] [Q&A]
Dear Dr. Tadano,

I've made two calculations of IFCs up to 4th order with ENET that are similar. The only difference is that in one of them I fixed the 2nd order IFCs with the result from a calculation with pattern displacements (MODE = suggest, 6 structures displaced in 0.003 A). The DFSET has 500 randomly displaced structures (0.04 A) and 330 structures with pattern displacements for 3rd order, displaced in 0.04 A.

The one with fixed 2nd order IFCs gave a residual error of 51.5%, and the other (adjusting IFCs of all orders) gave an error of 28.5%. Is it common this huge difference in the errors between the two methods?

Thanks in advance,
Ricardo.

**Comment by Lamed-git** [comment]
Perhaps you can refer to the first question in https://alamode.readthedocs.io/en/latest/faq.html and reduce the residual of the 2nd order force by using “--offset”.

**Reply by RicardoCecconello** [reply]
I'm already imposing the offset.

**Reply by ttadano** [reply]
Did you check if the quality of the second-order force constant is good?

Since you used a quite small displacement (0.003 A) for the harmonic calculations, 
the second-order force constants are influenced more sensitively by numerical noises.
